int16[2] centerPoint
int16[2] generalizedEntityPoint

